packages/eslint-config/index.js

```diff
@@ -1,8 +1,7 @@
 // This is the ESLint configuation used by Redwood projects.
 
 module.exports = {
-  extends: './shared.js',
-
+  extends: ['./shared.js', 'plugin:jsx-a11y/recommended'],
```

Comment:
Should the user be able to turn off the linting rules if the config.web.a11y is false?

Reply:
Yeah shouldn't have reverted it, was getting this weird eslint error; let me change it back then I'll ask for help on that. It was looking for the `redwood.toml`

Reply:
It seems to actually try to use the getConfig function?

![image](https://user-images.githubusercontent.com/32992335/112043567-0b43f600-8b06-11eb-8ca5-142617dcce4f.png)

Reply:
I think it's because the repo's `.estlinrc.js` extends the one meant for redwood apps:
https://github.com/redwoodjs/redwood/blob/9e2cca7d153b44ed30382f995180f8d673d23702/.eslintrc.js#L2

Reply:
Ah, I see, let me try and figure out why this is hapenning

Reply:
Resolved by explicitly setting the __REDWOOD__CONFIG_PATH